docs/src/site/markdown/index.md.vm

```diff
@@ -0,0 +1,134 @@
+- **Reproducible Builds**: since version 22 of this parent POM, managed plugins versions are expected to be compliant with [Reproducible Builds](https://reproducible-builds.org/)\.  
+    To choose their own release timestamp in output archives, projects using this POM as parent should override with a property in their root POM:
+    ```unknown
+      <properties>
+        <project.build.outputTimestamp>10</project.build.outputTimestamp>
+      </properties>
+    ```
+    The value will be updated by Maven Release Plugin during releases\. If a project wants to disable Reproducible Builds, just define the property value with any single non\-numeric character\.
+
+${esc.h}${esc.h} The `apache-release` Profile
+
+As noted above, this pom configures the release plugin to enable the `apache-release` profile for all executions of the release plugin\.
+
+The profile includes the following plugins:
+
+- maven\-assembly\-plugin  
+    [org\.apache\.apache\.resources:apache\-source\-release\-assembly\-descriptor:$context.get("version.apache-resource-bundles")](/apache-resource-bundles/source-release/) is added as dependency, and an execution is configured with `source-release-assembly` id\.  
+    The plugin is configured to take a `descriptorRef` name from the **sourceReleaseAssemblyDescriptor** property, which by default is set to `source-release` \(zip only\): you can override the property with `source-release-zip-tar` \(both zip and tar\) or `source-release-tar` \(tar only\) values\.  
+    If you want to avoid this default assembly execution to provide your own mechanism to produce the source release archive, you can disable the execution by configuring `skipAssembly` parameter for this `source-release-assembly` execution id
+
+- maven\-source\-plugin  
+    Configured to build and attach a source jar\.
+
+- maven\-javadoc\-plugin  
+    Configured to build and attach a javadoc jar\.
+
+- maven\-gpg\-plugin  
+    Configured to sign everything\.
+
+    On developer workstations it expects GnuPG agent to be available to get passphrases, while on fully unattended \(CI\-like\) workflows the use of `MAVEN_GPG_PASSPHRASE` environment variable use is recommended\.
+
+- checksum\-maven\-plugin  
+    Configured to create a checksum file\(s\) for source release as required by [Apache release distribution policy](http://www.apache.org/dev/release-distribution#sigs-and-sums)\.
+
+When doing a release with maven\-release\-plugin, this creates files in `target/checkout/target` ready to be copied to projects&apos; [Apache `/dist/` release distribution](http://www.apache.org/dev/release-distribution) directory:
+
+- `${artifactId}-${version}-source-release.[zip|tar.gz]`: the source release archive\(s\)
+- `${artifactId}-${version}-source-release.[zip|tar.gz].sha512`: their checksum\(s\)
+- `${artifactId}-${version}-source-release.[zip|tar.gz].asc`: their signature\(s\)
```

Comment:
`${artifactId}`, `${version}` should be escaped?

Reply:
Not necessary